packages/spruce-next-helpers/src/skillskit/next/_error.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {
+	Page,
+	PageContent,
+	Layout,
+	LayoutSection,
+	CardBuilder,
+	Card,
+	Button
+} from '@sprucelabs/react-heartwood-components'
+
+import { skill } from '@sprucelabs/spruce-next-helpers'
+
+export default class Error extends React.Component {
+	static getInitialProps({ res, err }) {
+		const statusCode = res ? res.statusCode : err ? err.statusCode : null
+		return { statusCode }
+	}
+
+	componentDidMount() {
+		skill.ready() // Show the skill
+	}
+
+	render() {
+		const { errorMessage, errorCTA } = this.props
+
+		let message =
+			this.props.statusCode && this.props.statusCode === 404
```

Comment:
Hrm; how can we lang this? 🤔

Reply:
We have a major lang issues right now. We need to really bring the lang from ai-web into skills and then update here.

Reply:
RFC after release 3